### PR TITLE
Pinmap for Intel Edison Sparkfun and documentation

### PR DESCRIPTION
**Pull request by devfubar**
This should fix issues #262 and #317

### CODE REVIEW

File: platforms/intel-iot/edison/sparkfun_pinmap.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
+package edison
+
+var sparkfunPinMap = map[string]sysfsPin{
+  "12": sysfsPin{
```
Comment (deadprogram) [inline platforms/intel-iot/edison/sparkfun_pinmap.go]: Shouldn't this be labeled "GP12" to exactly match the labels on the board? Likewise with the reset of the pins.

Comment (devfubar) [inline platforms/intel-iot/edison/sparkfun_pinmap.go]: I did think about that as it is what is on the board it self.  However most of the documentation including a lot of the Intel stuff references just the port number eg `echo 14 > /sys/class/gpio/export`.  When I came to try the code for the first time my natural instinct was to use "14" so thats what swayed it in my mind.....

Comment (deadprogram) [inline platforms/intel-iot/edison/sparkfun_pinmap.go]: I get what you are saying, seems like a good thing. 

At some point I think I will implement something to strip off unneeded but nice looking alpha prefixes, just to match board labeling, but that is entirely separate.

**Comment by deadprogram** [comment]
Great stuff, @devfubar thank you.

**Comment by devfubar** [comment]
No problem @deadprogram happy to help especially since you guys are saving em a lot of time :-)  I may add a few sensors and such as I go through, currently working on srf05 as I have a few of them kicking about...

**Pull request merged**